### ISSUE TITLE
rename aiida/cmdline/commands/code.py to something else

### ISSUE THREAD

**Issue by DropD** [aiidateam/aiida_core#1731]
## Problem 
In `aiida/cmdline/commands/shell.py` there is a statement `import code` which is meant to import the builtin module `code`. This works as long as `verdi shell --plain` is called outside the `commands` folder. However linters will rightfully complain that this is ambiguous or just assume that we are trying to import `aiida.cmdline.commands.code`.

## Proposed solution
Refactor: rename `aiida/cmdline/commands/code.py -> aiida/cmdline/commands/verdi_code.py`.

**Comment by giovannipizzi** [comment]
ok for me to rename

**Comment by sphuber** [comment]
I second this, because then we can also rename `importfile` to just `*_import` for consistency. Likewise, I would propose to prefix the actual sub commands in the command file with a common prefix, because given that they are top level functions, the desired name may also clash with reserved names. So:

* Rename code files to `verdi_command.py`
* Rename all subcommand functions to `cmd_name`, where name is the actual command name. I.e. for `verdi_calculation.py` you will have `cmd_list`, `cmd_kill` etc.

Importing and using the commands will look like:
```
from aiida.cmdline.commands import verdi_calculation

invoke(verdi_calculation.cmd_list)
```

**Comment by sphuber** [comment]
Fixed in PR #1786
